src/acceptance-tests/deploy/deploy_single_node_test.go

```diff
@@ -9,10 +10,6 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func createDeployment(postgresReleaseVersion int, manifestPath string, prefix string) error {
-	name := helpers.GenerateEnvName(prefix)
-	return updateDeployment(postgresReleaseVersion, manifestPath, name)
-}
 func updateDeployment(postgresReleaseVersion int, manifestPath string, name string) error {
```

Comment:
Can we call this `CreateOrUpdateDeployment ` to express the actual functionality?